src/Comps/cards/Cards.tsx

```diff
@@ -0,0 +1,387 @@
+import React from "react";
+import Card from "../card/Card";
+import { CardsContainer } from "./style";
+
+const Cards = () => {
+  const data = {
```

Comment:
This mock data will eventually return from an api.
Try to prepare your component as best as you can for that.
Meaning that you'll probably have a useEffect that calls the api and returns this data and saves it in a state.
Then you would probably map through this state and render your data.

Also try to keep your code as lean as possible and extract your mock data to an external file